src/main/_docs/integration-guide/integration-continuous-integration.md

```diff
@@ -52,66 +67,68 @@ You need a Codenvy Factory configured to use the project you want associated wit
         }
 ```
 
-### Set Codenvy Environment Variables
+## Set Codenvy Environment Variables
 
-#### Credentials
+### Credentials
 
-Update the `codenvy.env` with the username and password of the user who created the Factory in Codenvy. There can be just one user for all Factories used in the integration flow:
+Update the `codenvy.env` with the username and password of the user who created the Factory in Codenvy. You only need one user for all the Factories used by your Jenkins integration:
 
 ```text
 CODENVY_INTEGRATION_FACTORY_OWNER_USERNAME=somebody@somemail.com
 CODENVY_INTEGRATION_FACTORY_OWNER_PASSWORD=password
 ```
 
-#### Git
-**For GitHub**
-Update the `codenvy.env` with `GitHub` webhooks properties. Note that you can rename "WEBHOOK1" with any identifier. Webhooks are tied to connectors through Factory IDs. There can be multiple webhooks (if you need to receive webhooks from multiple repositories), i.e. a new group of webhook properties can be created. In this case, change identifier WEBHOOKID and FACTORY1_ID, which can be any strings.
+### Git
+The Jenkins integration supports both GitHub and BitBucket Server repos.
+
+#### For GitHub
+
+Update the `codenvy.env` with `GitHub` webhooks properties. Note that you can rename "WEBHOOK1" with any identifier. Webhooks are tied to connectors through Factory IDs. There can be multiple webhooks (if you need to receive webhooks from multiple repositories) - in this case, add a new entry with a different name for WEBHOOKID and FACTORY1_ID.
```

Comment:
>in this case, add a new entry with a different name for WEBHOOKID and FACTORY1_ID.

I would rephrase this:
in this case, add a new group of entries with a different name for WEBHOOKID.

FACTORY1_ID is described below